internal/cri/server/restart.go

```diff
@@ -190,7 +190,7 @@ func (c *criService) recover(ctx context.Context) error {
 	if err := eg.Wait(); err != nil {
 		return err
 	}
-
+	close(c.containerStore.Added)
```

Comment:
Thanks for filing this pull request.
I found there are several similar changes related to this topic. Would you please just focus on one of them?
It's easy for reviewers to track history comments, like https://github.com/containerd/containerd/pull/10409#discussion_r1719852537.

Thanks

Reply:
done, thank you, your idea is better. @fuweid

Reply:
The it run successfully. can you review this pr ? @fuweid

Reply:
ping @fuweid